.github/ISSUE_TEMPLATE.md

```diff
@@ -19,13 +12,9 @@ Please provide any relevant information about your setup. This is important in c
 * OpenMined Unity App Version:
 
 
-## Expected Behavior
-
-Please describe the behavior you are expecting
-
-## Current Behavior
+## Description
 
-What is the current behavior?
+Please describe the behavior you are expecting and the current behavior?
```

Comment:
This could be rephrased -- maybe "Please describe expected behavior and how it differs from current behavior."